### PR TITLE
Making the label formatter configurable

### LINKED ISSUE #2260: Option to show full class name in completion items
Currently when completing it shows only some parts.

It would (for me) be useful to show the full class.

Screenshot 2023-05-07 at 09 41 47

### PR DESCRIPTION
**Pull request by mamazu**
Closes #2260

### CODE REVIEW

File: lib/Extension/Completion/CompletionExtension.php [hunk 1]
```diff
@@ -99,7 +111,14 @@ private function registerCompletion(ContainerBuilder $container): void
         });
 
         $container->register(LabelFormatter::class, function (Container $container) {
-            return new HelpfulLabelFormatter();
+            switch ($formatter = $container->parameter(self::PARAM_LABEL_FORMATTER)->string()) {
```
Comment (dantleech) [inline lib/Extension/Completion/CompletionExtension.php:114]: could use `return match ($formatter) {` here

Comment (mamazu) [reply lib/Extension/Completion/CompletionExtension.php]: Yeah of course.

**Comment by mamazu** [comment]
Is that a flaky test?

**Review by dantleech — APPROVED** [review]

**Comment by dantleech** [comment]
yes it is, thanks :tada:

**Pull request merged**